app/controllers/homepage_controller.rb

```diff
@@ -51,21 +51,21 @@ def index
         raise ArgumentError.new("Unknown view_type #{@view_type}")
       end
 
+    main_search = location_search_available ? MarketplaceService::API::Api.configurations.get(community_id: @current_community.id).data[:main_search] : :keyword
+    location_search_in_use = location_search_available && main_search == :location
```

Comment:
The `location_search_available` is redundant in this line. The previous line takes that already into account.